x/cronos/spec/04_messages.md

```diff
@@ -0,0 +1,36 @@
+<!-- order: 4 -->
+
+# Messages
```

Comment:
one useful information for `MsgConvertVouchers` and `MsgTransferTokens` would be to specify from/to addresses which ecosystem they are expected to come from... I assume `address` in `MsgConvertVouchers` would the be used in EVM, so it should come from the Ethereum ecosystem, but when on first transfers in the IBC tokens, they may land on an address that comes from the Cosmos ecosystem?

or would those message check the account types?